### PR TITLE
Update to polkadot-v0.9.43

### PR DESCRIPTION
**Pull request by Agusrodri**
Resumes previous work from #922. Continues updating dependencies and fixing related bugs.

### Upstream related changes

* Removal of UMP pallet in favor of Message Queue pallet: https://github.com/paritytech/polkadot/pull/6271
* Addition of `MaxRemoteLockConsumers` and `RemoteLockConsumerIdentifier` types in `pallet-xcm` config https://github.com/paritytech/polkadot/pull/6947

### CODE REVIEW

**Comment by Agusrodri** [comment]
@xlc

**Review by xlc — APPROVED** [review]

**Comment by xlc** [comment]
can you make sure CI passes

File: xtokens/Cargo.toml [hunk 1]
```diff
@@ -13,41 +13,42 @@ scale-info = { version = "2.1.2", default-features = false, features = ["derive"
...
-frame-system = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.42" }
-sp-io = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.42" }
-sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.42" }
-sp-std = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.42" }
+frame-support = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.43" }
+frame-system = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.43" }
+sp-io = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.43" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.43" }
+sp-std = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.43" }
+pallet-message-queue = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.43" }
```
Comment (xlc) [inline xtokens/Cargo.toml:21]:
```suggestion
pallet-message-queue = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.43" }
```

and corresponding std feature flags

Comment (xlc) [reply xtokens/Cargo.toml:21]: actually, this should be in dev dependencies?

Comment (Agusrodri) [reply xtokens/Cargo.toml:21]: Yes I think it should, I put it in normal deps at the beginning but then I only modified test-related files, so I will change it

**Comment by xlc** [comment]
release branch created https://github.com/open-web3-stack/open-runtime-module-library/tree/polkadot-v0.9.43

**Comment by Billiej88** [comment]
https://github.com/open-web3-stack/open-runtime-module-library/pull/928#issuecomment-1603454315

**Comment by Billiej88** [comment]
Moonsong-Labs:update-polkadot-v0.9.43

**Comment by Billiej88** [comment]
Moonsong-Labs:update-polkadot-v0.9.43

**Pull request merged**